apps/api_web/lib/api_web/controllers/service_controller.ex

```diff
@@ -163,6 +163,29 @@ defmodule ApiWeb.ServiceController do
               enum: Enum.to_list(0..5),
               example: 1
             )
+
+            rating_start_date(
+              :string,
+              "Earliest date which is a part of the rating (season) which contains this service. Format is ISO8601.",
+              "x-nullable": true,
```

Comment:
Is this actually nullable? I thought they all had start dates?

Reply:
Currently Foxboro services don't, but we could add one if it made sense.

Separately, it looks like services created by disruptions aren't getting rating columns filled in. I'll file a ticket to fix that.

Reply:
Fair enough, re: Foxboro. I retract my comment.